test/ParamsTabExpansion.Tests.ps1

```diff
@@ -0,0 +1,45 @@
+. $PSScriptRoot\Shared.ps1

```

Comment:
Can you rename this to `GitParamTabExpansion.Tests.ps1`.  We have a fledgling conventions of `<filename-under-test>.Tests.ps1`.  Thanks.